apps/glances/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3.7"
+services:
+    glances:
+        container_name: glances
+        restart: unless-stopped
+        ports:
+            - ${APP_PORT}:61208
+            - 8421:61209
```

Comment:
For what is this port needed? I removed it and app launched the same

Reply:
Honestly, not particularly sure- Steve and I just followed the docs. If it works without it, then there's no need for it. Feel free to remove it!